### PR TITLE
Update upgrade_messages.dart

### PR DESCRIPTION
**Pull request by alhamri**
minor change in Arabic translating of the word "later".

### CODE REVIEW

File: lib/src/upgrade_messages.dart [hunk 1]
```diff
@@ -335,7 +335,10 @@ class UpgraderMessages {
     String message;
     switch (languageCode) {
       case 'ar':
-        message = 'لاحقاً';
+        //only minor change here, removing the character at the left top of the arabic word (لاحقاً)
+        // before: message = 'لاحقاً';
+        //now:
+        message = 'لاحقا';
```
Comment (larryaasen) [inline lib/src/upgrade_messages.dart:341]: This looks the same to me. The new line 341 looks the same as the old line 338. Even a Find for `لاحقا` returns both results on this page.

Comment (alhamri) [reply lib/src/upgrade_messages.dart:341]: Yes they are look the same, but they are not. The previous one contains the (Tanween) character (` ً `) `لاحقاً` , I ran it on emulators and on physical devices and it gave me the same result with uncomfortable appearance:

later1
later2

but after I remove this character it worked fine with the new word (`لاحقا`):
later3
later4

This character is not necessary in Arabic language to write it in a single word.

Comment (larryaasen) [reply lib/src/upgrade_messages.dart:341]: I see now. Sorry it took me so long to see this.

Comment (alhamri) [reply lib/src/upgrade_messages.dart:341]: no problem mate, and thank you for sharing us your awesome plugin

Comment (larryaasen) [reply lib/src/upgrade_messages.dart:341]: This looks great. I will merge now.

**Pull request merged**